tests/storage/data_import_cron/test_data_import_cron_pvc_source.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+import pytest
+
+from tests.storage.data_import_cron.utils import (
+    wait_for_new_data_import_cron_re_import,
+    wait_for_new_pvc_uid_after_delete_import,
+)
+
+LOGGER = logging.getLogger(__name__)
+
+
+class TestDataImportCronPvcSource:
+    @pytest.mark.polarion("CNV-11842")
+    def test_data_import_cron_with_pvc_source_ready(
+        self, namespace, dv_for_data_import_cron, data_import_cron_pvc_source, vm_for_data_import_cron
+    ):
+        pvc_uid = dv_for_data_import_cron.pvc.instance.metadata.uid
+        digest_full = data_import_cron_pvc_source.instance.status.currentImports[0].Digest
+        digest_uid = digest_full.split("uid:")[1]  # Extract just the UUID part
+        assert pvc_uid == digest_uid, f"PVC UID {pvc_uid} does not match DataImportCron Digest {digest_uid}"
+
+    @pytest.mark.polarion("CNV-11858")
+    def test_data_import_cron_with_pvc_source_re_trigger_import(
+        self, namespace, dv_for_data_import_cron, data_import_cron_pvc_source, vm_for_data_import_cron
+    ):
+        pre_delete_pvc_uid = dv_for_data_import_cron.pvc.instance.metadata.uid
+        post_delete_pvc_uid = wait_for_new_pvc_uid_after_delete_import(data_source=dv_for_data_import_cron.pvc)
+        data_cron_digest_uid = wait_for_new_data_import_cron_re_import(
+            orig_digit=pre_delete_pvc_uid, data_import_cron=data_import_cron_pvc_source
+        )
```

Comment:
I don't get it. 
What is the user flow for this feature? 
Do they must delete the source PVC in order to trigger a new import? Or can they just write some data to that PVC?